### PR TITLE
better buffer instructions and other minor changes

### PR DESCRIPTION
**Pull request by mscurtescu**
- the `sysctl` changes can be applied on the host
  - relax warnings
  - better instructions for the host
  - how to detect buffer size issues
- add git commit to image as a label
  - updated build command sample to provide git commit
- add resource requirements
  - docker compose is not enforcing them
- add default stats collection
- fix several config file permissions

### CODE REVIEW

**Comment by mscurtescu** [comment]
Related PR: https://github.com/QuilibriumNetwork/ceremonyclient/pull/66

Adds section to DOCKER-README.md explaining docker resource management.

**Comment by mscurtescu** [comment]
Thank you @sercanCyberVision for the resource management section!

File: DOCKER-README.md [hunk 1]
```diff
@@ -3,39 +3,65 @@
...
 ## Run
 
+You can run Quilibrium on the same machine where you built the image, from the same repository root
+folder where [docker-compose.yml](docker-compose.yml) is.
+
+You can also copy `docker-compose.yml` to a new folder on a server and run it there. In this case you
+have to have a way to push your image to a Docker image repo and then pull that image on the server.
+Github offers such an image repo and a way to push and pull images using special authentication
+tokens. See
+[Working with the Container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry).
```
Comment (mscurtescu) [inline DOCKER-README.md:63]: @CassOnMars and plans to start pushing images to Docker Hub? Happy to help with that, let me know.

FYI working on building a much leaner image, in a followup PR.

File: docker-compose.yml [hunk 2]
```diff
@@ -3,16 +3,24 @@ version: "3.8"
...
     build: ./
     image: quilibrium
     restart: unless-stopped
+    deploy:
+      resources:
+        limits:
+          memory: 32G
+        reservations:
+          cpus: 12
+          memory: 16G
```
Comment (mscurtescu) [inline docker-compose.yml:19]: @CassOnMars do these resources look reasonable?

Is there a hard coded limit on number of CPUs?

Comment (Freekers) [reply docker-compose.yml:19]: For Dawn, I got out of memory even with 32GB

Comment (mscurtescu) [reply docker-compose.yml:19]: > For Dawn, I got out of memory even with 32GB

With the above config? Trying to determine if 32GB is not enough for Quilibrium and Docker is not allowing more, or Quilibrium keeps taking more that 32GB and the host runs out of memory?

Comment (Freekers) [reply docker-compose.yml:19]: > > For Dawn, I got out of memory even with 32GB
> 
> With the above config? Trying to determine if 32GB is not enough for Quilibrium and Docker is not allowing more, or Quilibrium keeps taking more that 32GB and the host runs out of memory?

No, not with the config above. I'm currently running Quilibrium natively using Go. Quilibrium seems to keep taking more than 32GB RAM and the host runs out of memory. Happens after the node has been running for about 4 hours

File: DOCKER-README.md (hunk 1 above)
Comment (CassOnMars) [reply DOCKER-README.md:63]: no immediate plans but if you want to take point on this i'm supportive!

File: docker-compose.yml (hunk 2 above)
Comment (CassOnMars) [reply docker-compose.yml:19]: 16 is the minimum for Dawn, going OOM is a bug

**Review by CassOnMars — APPROVED** [review]

**Pull request merged**